src/appmixer/hubspot/bundle.json

```diff
@@ -42,6 +42,9 @@
         "2.2.0": [
             "New component: Engagements - GetEmails: Read a batch of emails by internal ID, or unique property values.",
             "Update component: GetContact: fix support for adding properties, add support for retrieving associations."
+        ],
+        "3.0.0": [
+            "(breaking change) Improve webhook handling by better event distribution. Affected components: NewContact, UpdatedContact, NewDeal, UpdatedDeal."
```

Comment:
@vtalas It is breaking because all the trigger components need to be stopped and started again in order to use the keys in `context.store` with `portalId`.

Reply:
One thing. We want to rewrite this to use the new `events/listeners`. 
We discussed with @vtalas that we want to fix it first for version 5, ideally without a breaking change. But that may not be possible. If it is not possible to do it without the breaking change, then I would not release this, until we have the version with `events/listeners`. The reason:
- A customer (even a customer already on version 6) will update the Hubspot, and all the flows have to be started again. 
- Then, in a week we will release another breaking version that works with the `events/listener` and the customer will have to install a new major version again. 

So either write directly a version with `events/listeners`, or prepare both and release `3.0.0` without dependency on Appmixer >=6 and 3.0.1 with dependency on Appmixer >= 6

Reply:
@martin-krcmar This is a fix for Appmixer <6 only (no `events/listeners` support). It is a breaking change as mentioned above.

What should be the next steps then? I can imagine something like this
- Hubspot 3.0.0 (this PR) merged into `dev`
- Hubspot 4.0.0 (another PR) with this:
    - rewrite 3.0.0 to `events/listeners`
    - for Appmixer >=6 only
    - merge into `dev`
- Release both versions at the same time